core/src/test/java/dev/streamx/cli/OsUtils.java

```diff
@@ -0,0 +1,8 @@
+package dev.streamx.cli;
+
+import com.github.jknack.handlebars.internal.text.StringEscapeUtils;
```

Comment:
quite strange import, isn't it? maybe we can copy this logic if it's small or use some more dedicated utils?